tools/alt-testing/create-projects.sh

```diff
@@ -0,0 +1,43 @@
+set -e
```

Comment:
this file should be marked as executable as well, suggest running `git update-index --chmod=+x tools/alt-testing/create-projects.sh`

Reply:
done